### PR TITLE
I AM A CODER I AM A CODER I AM A CODER 

### PR DESCRIPTION
**Pull request by Therandomhoboo**
[i am a coder](url)

We need this for all our coders out there, so we can tell Baimou we are indeed coders [Updated]

### CODE REVIEW

File: src/main/java/net/yogstation/yogbot/listeners/commands/image/CoderCommand.kt [hunk 1]
```diff
@@ -72,6 +72,7 @@ class CoderCommand(discordConfig: DiscordConfig, random: Random) : ImageCommand(
 		"https://cdn.discordapp.com/attachments/734475284446707753/1103234273735090196/image.png", // Just get playtime
 		"https://cdn.discordapp.com/attachments/408008557993132035/1161765671085023403/image.png", // I'm normal I swear
 		"https://cdn.discordapp.com/attachments/892167119154872391/1161764164788830328/and_if_we_help.png" // Helping out
```
Comment (adamsong) [inline src/main/java/net/yogstation/yogbot/listeners/commands/image/CoderCommand.kt]: Forgot a comma

**Review by adamsong — CHANGES_REQUESTED** [review]
Image kinda meh

File: src/main/java/net/yogstation/yogbot/listeners/commands/image/CoderCommand.kt (hunk 1 above)
Comment (Therandomhoboo) [reply src/main/java/net/yogstation/yogbot/listeners/commands/image/CoderCommand.kt]: Doneskis

**Comment by Therandomhoboo** [comment]
> Image kinda meh

Thank you, I didn't want it to be too good otherwise people would ask me to do more.

**Comment by Therandomhoboo** [comment]
> Image kinda meh

RIP

**Pull request merged**